guides/release/components/conditional-content.md

```diff
@@ -105,7 +105,7 @@ is passed in and is truthy.
 <aside ...attributes>
   <div
     class="avatar {{if @isActive "is-active"}}"
-    title="{{@title}}"
+    title={{@title}}
```

Comment:
I'd probably still keep these quotes personally. I'm curious, what made you remove them?

Reply:
Wouldn't this fail `no-unnecessary-concat`?